setup.sh

```
@@ -0,0 +1,111 @@
+  esac
+done
+
+echo "[SAOS] 🔥 Initializing Sovereign CLI Forge..."
+
+# --- 1. ROOTLESS DOCKER ---
+echo "[SAOS] Installing Rootless Docker..."
+curl -fsSL https://get.docker.com/rootless | sh
+
+echo 'export PATH=$HOME/bin:$PATH' >> ~/.bashrc
+echo 'export DOCKER_HOST=unix://$XDG_RUNTIME_DIR/docker.sock' >> ~/.bashrc
+source ~/.bashrc
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)